src/index.js

```diff
@@ -1,8 +1,32 @@
 const https = require('https');
 
 const MANIFEST_URL = 'https://raw.githubusercontent.com/apify/actor-templates/master/templates/manifest.json';
+const CONSOLE_README_SUFFIX_URL = 'https://raw.githubusercontent.com/apify/actor-templates/master/templates/consoleReadmeSuffix.md';
+const LOCAL_README_SUFFIX_URL = 'https://raw.githubusercontent.com/apify/actor-templates/master/templates/localReadmeSuffix.md';
+
+/**
+ * This will fetches the readme suffixes for the console and local templates.
+ * These suffixes are then added to the manifest object at the root level
+ */
+async function fetchReadmeSuffixes() {
+    let consoleReadmeSuffix = '';
+    let localReadmeSuffix = '';
+
+    try {
+        const consoleResponse = await fetch(CONSOLE_README_SUFFIX_URL);
+        const localResponse = await fetch(LOCAL_README_SUFFIX_URL);
```

Comment:
We can't use `fetch` unless we bump CLI's requirement to Node 18, which I don't think we should do.

<img width="767" alt="CleanShot 2023-06-07 at 14 18 06@2x" src="https://github.com/apify/actor-templates/assets/23726914/9a099bb4-c2c9-4846-abb2-8bc69421e398">

Reply:
We could use https://github.com/nodejs/undici